### ISSUE TITLE
Checking output folder in picca_delta_extraction

### ISSUE THREAD

**Issue by andreicuceu** [igmhub/picca#937]
When running picca_delta_extraction this code is used to initialize output folders and check for previous runs (L609-631 of config.py).
```
def initialize_folders(self):
        """Initialize output folders
        Raise
        -----
        ConfigError if the output path was already used and the
        overwrite is not selected
        """
        if not os.path.exists(self.out_dir):
            os.makedirs(self.out_dir, exist_ok=True)
            os.makedirs(self.out_dir + "Delta/", exist_ok=True)
            os.makedirs(self.out_dir + "Log/", exist_ok=True)
            self.write_config()
        elif self.overwrite:
            os.makedirs(self.out_dir + "Delta/", exist_ok=True)
            os.makedirs(self.out_dir + "Log/", exist_ok=True)
            self.write_config()
        else:
            raise ConfigError("Specified folder contains a previous run. "
                              "Pass overwrite option in configuration file "
                              "in order to ignore the previous run or "
                              "change the output path variable to point "
                              f"elsewhere. Folder: {self.out_dir}")
```

However, this doesn't actually check for previous runs. It just checks if the output folder exists or not, and if it already exists it throws an error even if the folder is empty (unless the overwrite flag is passed). We should change this to check if there is actually anything in the folder.

**Comment by iprafols** [comment]
The method write_config also writes the configuration file .config.ini, so it "initializes" the picca run in the folder it's creating. This is why this function is only checking for the folder's existence. However, I agree that this could be improved. We could, for example, create the folder manually and then try to run on it. I'll modify so that it checks for the .config.ini file.

**Comment by iprafols** [comment]
another question is whether or not we should rename the .config.ini to config.ini so that it is clearer that the folder is not empty

**Comment by iprafols** [comment]
@andreicuceu, this issue was addressed by PR #942. If you agree, we will close the issue

**Comment by andreicuceu** [comment]
Looks good, thanks Ignasi!
